### DISCUSSION TITLE
Still liking lit-html and tailwind the most

### DISCUSSION THREAD

**Discussion by SuperstringDev** [lit/lit#3961] [General]
For my own projects based on a custom framework I'm happy to just use lit-html and not lit-element. After looking around for anything related to using tailwind with lit-html I'm not finding anything but what I am finding is a few articles with huge work arounds just to work outside the virtual DOM that lit-elements use.

Since lit-html has nothing to do with web components would there be any issues using lit-html and tailwind together? Any heads up on potential problems are appreciated.

**Comment by augustjk** [comment]
I have no personal experience with using tailwind but my understanding of it is that it's based on tooling that scans what classes you're adding to templates. If it can pick up class names used in the `html` tagged templates, it should be fine.

**Comment by AnNingUI** [comment]
I think [twind](https://twind.style/packages/@twind/core) is more suitable than tailwind in lit, even though it's not as perfect as tailwind, it's really much more convenient to generate styles dynamically through js than tailwind even in web components shadow dom
